request.d.ts

```diff
@@ -0,0 +1,6 @@
+import { PinejsClientCore } from './core'
+import * as request from 'request'
```

Comment:
I'm having an issue where when I try `tsc` to validate this it appears to be trying to use types from `./request` and not the `request` module from npm? I don't know if it will affect downstream users or not but if you guys have any idea on that it'd be great